esp/src/eclwatch/WUQueryWidget.js

```diff
@@ -456,6 +457,13 @@ define([
             this.workunitsGrid.onSelectionChanged(function (event) {
                 context.refreshActionState();
             });
+
+            aspect.after(this.workunitsGrid, 'gotoPage', function (promise, args) {
```

Comment:
This is correct as well but we must pass in the second argument (deferred, **args**)